packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -288,11 +288,21 @@ export class FluidDataStoreRuntime extends EventEmitter implements IFluidDataSto
 
         // Check for a data type reference first
         if (this.contextsDeferred.has(id) && parser.isLeaf(1)) {
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            const value = await this.contextsDeferred.get(id)!.promise;
-            const channel = await value.getChannel();
-
-            return { mimeType: "fluid/object", status: 200, value: channel };
+            try {
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                const value = await this.contextsDeferred.get(id)!.promise;
+                const channel = await value.getChannel();
+
+                return { mimeType: "fluid/object", status: 200, value: channel };
+            } catch (error) {
+                this.logger.sendErrorEvent({ eventName: "GetChannelFailedInRequest", error: "error" }, error);
```

Comment:
remove error: "error" - this is not needed, sendErrorEvent will do the right thing